map.js

```diff
@@ -0,0 +1,46 @@
+(function() {
+	'use strict';
+	let mymap;
```

Comment:
Define `mymap` inside the callback of `kintone.events.on(event, callback)`. Then you can change `let` to `const` as the value doesn't need to be updated anymore.
For example, you can change this

```js
    let mymap;
    kintone.events.on('app.record.detail.show', function(event) {
        const record = event.record;
        // ...

        mymap = L.map('nihonbashi').setView(cybozuAddress, 16);
```

to the following.

```js
    kintone.events.on('app.record.detail.show', function(event) {
        const record = event.record;
        // ...

        const mymap = L.map('nihonbashi').setView(cybozuAddress, 16);
```
If you want to pass `mymap` to the `setMarker()` function, you can pass it as an argument.